hw/dv/dpi/dmidpi/dmidpi.c

```diff
@@ -31,7 +31,11 @@ const int IDCODEVAL = 0x04F5484D;
 // [3:0]    0x1  - Protocol version (0.13)
 const int DTMCSRVAL = 0x00000071;
 
+#ifndef __cplusplus
```

Comment:
same here, do we need the underlying type for the enum? or can you just drop the `ifdef`s and the `: uint8_t` on these enums and everything still works?

Reply:
Can confirm works with my use case in C, will let CI tell the full story :)